components/forms/form-builder/FormArrayPanelBuilder.tsx

```diff
@@ -47,9 +53,9 @@ export function FormArrayPanelBuilder({
   const formattedFieldName = formatFieldName(field.name);
 
   return (
-    <>
+    <div id={field.name} data-cy={`${field.name}-panel`}>
       {formData[field.name]?.map((_arrObj: any, index: number) => (
-        <Card key={index} className="container">
+        <Card key={index} id={`${field.name}-${index}`} className="container">
```

Comment:
Tricky to get a unique id for this one with it being a generic panel builder. 
Might have to use `uuid` library to generate a unique panel id, then temporarily add it to the panel data for rendering purposes - which is a faff.
Note: we can't generate the uuid on the fly in the return as it will never match up between renders.